e2e/bare/client.py

```diff
@@ -23,6 +23,11 @@ def evaluate(self, parameters, config):
         accuracy = 1 - loss
         return loss, 1, {"accuracy": accuracy}
 
+def client_fn(cid):
+    _ = cid
```

Comment:
@charlesbvll maybe rather disable the linter error instead of this? I am guessing you are doing it to suppress the linter warning?